index.js

```diff
@@ -13,8 +13,8 @@ http.get(req, (res) => {
 
   res.on('end', () => {
     try {
-      let result = content.match(/meta property="og:image" content="(.*)"/);
-      result = result[0].match(/content="(.*)"/);
+      let result = content.match(/property="og:image" content="(.*)"/);
```

Comment:
It is important to get the right content, so we need a `meta` with those properties.

A RegExp that will better fit is `/<meta ([^>]*)property="og:image" content="([^\"]*)"`.

Can you please work based on it?